operator/src/test/java/oracle/kubernetes/operator/helpers/Matchers.java

```diff
@@ -165,29 +170,63 @@ public void describeTo(Description description) {
   }
 
   static class EnvVarMatcher extends TypeSafeDiagnosingMatcher<V1EnvVar> {
+    private static final String DONTCARE = "SENTINEL_DONT_CARE";
     private final String expectedName;
+    private final String expectedValueRegEx;
 
     private EnvVarMatcher(String expectedName) {
       this.expectedName = expectedName;
+      this.expectedValueRegEx = DONTCARE;
+    }
+
+    private EnvVarMatcher(String expectedName, String expectedValueRegEx) {
+      this.expectedName = expectedName;
+      this.expectedValueRegEx = expectedValueRegEx;
     }
 
     static EnvVarMatcher envVarWithName(@Nonnull String name) {
       return new EnvVarMatcher(name);
     }
 
+    static EnvVarMatcher envVarWithName(@Nonnull String name, String value) {
```

Comment:
The method name implies that it is only matching the name, but it is actually matching the name and value. It should be named in a way that makes it clear what is happening. Simple "envVar" would probably be a more readable choice.

Reply:
Changed to hasEnvVarRegEx(str,str), this corresponds with hasEnvVar(str,str) above...